docker_and_oci.rst

```diff
@@ -529,9 +536,10 @@ ensure that the credentials needed to access the image are available to
 A build might be run as the ``root`` user, e.g. via ``sudo``, or under
 your own account.
 
-If you are running the build as ``root``, using ``sudo``, then any
-stored credentials or environment variables must be available to the
-``root`` user:
+If you are running the build as ``root``, using ``sudo``, then any stored
```

Comment:
Change the comma before `using` to `or`

Reply:
thanks! updated.

Reply:
On second thought, it's really better to just delete the `or using sudo,` because at best it is redundant, and sudo can be used to become other users too.

Reply:
sure, removed.